generators/entity/index.js

```diff
@@ -199,8 +199,8 @@ module.exports = EntityGenerator.extend({
         try {
             this.fileData = this.fs.readJSON(this.fromPath);
         } catch (err) {
-            this.log(chalk.red('\nThe configuration file could not be read!\n'));
-            return;
+            this.log(chalk.red('\nThe entity configuration file could not be read!\n'));
+            throw err;
```

Comment:
Dont throw error. Use `this.error ()` instead of `this.log ()` and yeoman will exit automatically and thats the recommended approach